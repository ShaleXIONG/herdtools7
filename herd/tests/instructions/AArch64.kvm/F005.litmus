AArch64 F005
EL0=P0
{
0:X0=0;
}
  P0           | P0.F           ;
 ADD W0,W0,#1  | ADD W0,W0,#1   ;
 SVC #0        | ERET           ;
 ADD W0,W0,#1  |                ;
forall 0:X0=3