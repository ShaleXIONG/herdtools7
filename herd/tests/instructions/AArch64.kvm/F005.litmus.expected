Test F005 Required
States 1
0:X0=3;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X0=3)
Observation F005 Always 1 0
Hash=da368a7cbd892872d6419160eb88a5f0

